helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 appVersion: "1.0.0"
-name: aws-efs-csi-driver
 description: A Helm chart for AWS EFS CSI Driver
-version: 0.3.0
+name: aws-efs-csi-driver
+version: 0.4.0
```

Comment:
leave as 0.1.0 for now, since https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/292 merged and  the only chart in the index is 0.1.0. After https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/291 merges, I will try to bump the version and include this and everything else between 0.1.0 and now.